.circleci/config.yml

```diff
@@ -273,6 +276,8 @@ jobs:
           name: Run tests
           command: |
             . venv/bin/activate
+            echo "${GCP_SERVICE_ACCOUNT_JSON_BASE64:?}" | base64 -d > $HOME/gcloud-service-user.json
+            export GOOGLE_APPLICATION_CREDENTIALS=$HOME/gcloud-service-user.json
```

Comment:
- [ ] TODO: Can I pull some of this stuff into its own PR?  Maybe more options, fewer overriding of command?